### DISCUSSION TITLE
Refresh toggle button intent

### DISCUSSION THREAD

**Discussion by jfree** [bric3/fireplace#51] [General]
What is the intent of the `Refresh` toggle button? Is it just for debug? With it enabled, I notice that selecting a frame makes the expected frame selection highlighting, but then this is reset after a couple of seconds.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/1835893/161414815-3b3948ce-c2c1-4e12-8468-dff4ef5a95b5.png">

**Comment by bric3** [comment]
I made this as a way to refresh the UI when I needed to rebuild the component.
